src/ios-deploy/ios-deploy.m

```diff
@@ -658,7 +658,14 @@ void mount_developer_image(AMDeviceRef device) {
         
         on_error(@"Unable to mount developer disk image. (%x)", result);
     }
-
+	
+	CFStringRef symbols_path = copy_device_support_path(device, CFSTR("Symbols"));
+	if (symbols_path != NULL)
+	{
+		NSLogOut(@"Symbol Path: %@", symbols_path);
```

Comment:
Can you add a NSLogJSON() call so that this can be easily parsed with the `--json` flag?
```
NSLogOut(@"Symbol Path: %@", symbols_path);
NSLogJSON(@{@"Event": @"MountDeveloperImage",
            @"SymbolsPath": (__bridge NSString *)symbols_path
            });
```